backend/src/v5/utils/responder.js

```diff
@@ -148,7 +149,7 @@ Responder.writeStreamRespond = (req, res, resCode, readStream, fileName, fileSiz
 		logger.logInfo(genResponseLogging(response.code, fileSize, req));
 	});
 
-	res.writeHead(response.status, headers);
+	res.writeHead(response.status, deleteIfUndefined(headers));
```

Comment:
why do you need this? what have you added that may be undefined?

Reply:
Im not passing the size that why I added it